File: BryanDanielV_StudentPortfolioPlatform/scratch/fix_all_typos.js
const fs = require('fs');
const path = require('path');

const replacements = [
  [/\bslate-805\b/g, 'slate-800'],
  [/\bslate-880\b/g, 'slate-800'],
  [/\bslate-205\b/g, 'slate-200'],
  [/\bslate-505\b/g, 'slate-500'],
  [/\bslate-550\b/g, 'slate-500'],
  [/\bslate-655\b/g, 'slate-600'],
  [/\bslate-705\b/g, 'slate-700'],
  [/\bslate-355\b/g, 'slate-300'],
  [/\bred-605\b/g, 'red-600'],
  [/\bred-955\b/g, 'red-900'],
  [/\bgreen-655\b/g, 'green-600'],
  [/\bamber-955\b/g, 'amber-900'],
  [/\bblue-955\b/g, 'blue-900'],
  [/\bamber-808\b/g, 'amber-800']
];

const frontendDir = 'c:\\internship\\frontend\\src';

function processFile(filePath) {
  let content = fs.readFileSync(filePath, 'utf8');
  let original = content;
  
  for (const [pattern, replacement] of replacements) {
    content = content.replace(pattern, replacement);
  }
  
  if (content !== original) {
    fs.writeFileSync(filePath, content, 'utf8');
    console.log(`Updated: ${filePath}`);
  }
}

function walk(dir) {
  const files = fs.readdirSync(dir);
  for (const file of files) {
    const fullPath = path.join(dir, file);
    const stat = fs.statSync(fullPath);
    if (stat.isDirectory()) {
      walk(fullPath);
    } else if (file.endsWith('.tsx') || file.endsWith('.ts') || file.endsWith('.css')) {
      processFile(fullPath);
    }
  }
}

walk(frontendDir);

File: BryanDanielV_StudentPortfolioPlatform/scratch/fix_all_typos.py
import os
import re

replacements = {
    r'\bslate-805\b': 'slate-800',
    r'\bslate-880\b': 'slate-800',
    r'\bslate-205\b': 'slate-200',
    r'\bslate-505\b': 'slate-500',
    r'\bslate-550\b': 'slate-500',
    r'\bslate-655\b': 'slate-600',
    r'\bslate-705\b': 'slate-700',
    r'\bslate-355\b': 'slate-300',
    r'\bred-605\b': 'red-600',
    r'\bred-955\b': 'red-900',
    r'\bgreen-655\b': 'green-600',
    r'\bamber-955\b': 'amber-900',
    r'\bblue-955\b': 'blue-900',
    r'\bamber-808\b': 'amber-800'
}

frontend_dir = r"c:\internship\frontend\src"

def process_file(filepath):
    with open(filepath, 'r', encoding='utf-8') as f:
        content = f.read()
    
    original = content
    for pattern, replacement in replacements.items():
        content = re.sub(pattern, replacement, content)
        
    if content != original:
        with open(filepath, 'w', encoding='utf-8') as f:
            f.write(content)
        print(f"Updated: {filepath}")

for root, dirs, files in os.walk(frontend_dir):
    for file in files:
        if file.endswith(('.tsx', '.ts', '.css')):
            process_file(os.path.join(root, file))
